Gemfile

```diff
@@ -85,6 +85,7 @@ gem "recaptcha", "~> 5.5", require: "recaptcha/rails" # Helpers for the reCAPTCH
 gem "redcarpet", "~> 3.5" # A fast, safe and extensible Markdown to (X)HTML parser
 gem "redis", "~> 4.2.1" # Redis ruby client
 gem "redis-rails", "~> 5.0.2" # Redis for Ruby on Rails
+gem "redis-store", "~> 1.9.0" # Redis storage options for Rails (listed here to lock version above 1.9.0 - see https://github.com/redis-store/redis-store/pull/333)
```

Comment:
I don't think we need to add this to the Gemfile as it's not an explicit dependency, I ran `bundle update redis-store` on master and it did update it nonetheless.

Maybe we can move `see https://github.com/redis-store/redis-store/pull/333` and a short explanation about what's going on in the routes file

Reply:
I swear version Redis-store 1.9 wasn't updating for me, but now that 1.9 is in `Gemfile.lock` I agree this isn't needed in the Gemfile anymore! I'll move this PR link to routes per your suggestion.